### PR TITLE
init read me file

### PR DESCRIPTION
**Pull request by danny-010**
new pr from mac

### CODE REVIEW

**Comment by danny-niu** [comment]
pls add datetime

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+danny init file
```
Comment (danny-niu) [inline README.md:1]: add datetime

Comment (danny-010) [reply README.md:1]: ok, got it

**Review by danny-010 — COMMENTED** [review]
this is a  remind

File: README.md (hunk 1 above)
Comment (danny-010) [inline README.md:1]: add datetime

Comment (danny-niu) [reply README.md:1]: ok

**Review by danny-niu — APPROVED** [review]

**Pull request merged**